Packs/EmailCommunication/ReleaseNotes/2_0_9.md

```diff
@@ -0,0 +1,21 @@
+
+#### Scripts
+
+##### SummarizeEmailThreads
+
+- Updated the Docker image to: *demisto/python3:3.10.11.58677*.
+##### DisplayEmailHtmlThread
+
+- Updated the Docker image to: *demisto/python3:3.10.11.58677*.
+##### SendEmailReply
+
+- Added the argument *subject_include_incident_id* to decide if to include the Incident id within the email subject.
+- Updated the Docker image to: *demisto/python3:3.10.11.58677*.
+
+##### PreprocessEmail
+
+- Added the argument *CreateIncidentUntaggedEmail* to control whether to allow the creation for email that we can't find existing incident for it. 
+- Updated the Docker image to: *demisto/python3:3.10.11.58677*.
+##### DisplayEmailHtml
+
+- Updated the Docker image to: *demisto/python3:3.10.11.58677*.
```

Comment:
```suggestion
Updated the Docker image to: *demisto/python3:3.10.11.58677*.
```

Reply:
@ShirleyDenkberg this is the approved auto format for release notes with a `-` at the begining.
https://github.com/demisto/demisto-sdk/blob/47bac94b49646cd165adea7235461dd459b255b7/demisto_sdk/commands/update_release_notes/update_rn.py#L769